data/json/items/generic.json

```diff
@@ -107,6 +107,285 @@
     "to_hit": -2,
     "flags": [ "TRADER_AVOID" ]
   },
+  {
```

Comment:
There are enough of these parts and they're closely related enough that I think it would be helpful to move them all to a new data/json.items/robot_parts.json file.

Reply:
The schematics *could* also go there, but they fit better in book/electronics.json, so that's not as clearly good.